### PR TITLE
Add mkdir and rmdir commands, for adding and removing directories on …

### PR DESCRIPTION
**Pull request by scottfabini**
…remote server.

### CODE REVIEW

File: EIAClient.java [hunk 1]
```diff
@@ -38,8 +38,12 @@ public String getHost() {
     public boolean init(HashMap main) {
 
         commands.putAll(main);
-        commands.put("connect", () -> { connect(input); } );
+        commands.put("connect", () -> {
+            connect(input);
+        });
         commands.put("disconnect", () -> {disconnect(); } );
+        commands.put("mkdir", () -> {mkdir(input); } );
```
Comment (galxy25) [inline EIAClient.java]: :+1: that we can use a library so we don't need to reinvent the wheel here and can have one to one mappings between what the user sees and what is actually happening.

File: EIAClient.java [hunk 2]
```diff
@@ -183,4 +187,47 @@ public boolean ls() {
...
+            }
+            if (input.length != 2) {
+                System.out.println("Incorrect number of parameters for mkdir. Type 'help' for command syntax.");
+            }
+
+            return ftp.makeDirectory(input[1]);
+
+        }
+
+        catch (IOException e) {
```
Comment (galxy25) [inline EIAClient.java]: Is this the only type of exception we need to worry about? @scottfabini @pault2k14

File: EIAClientTest.java [hunk 3]
```diff
@@ -18,4 +18,15 @@ public void testDisconnect() throws Exception {
     public void testLs() throws Exception {
 
     }
```
Comment (galxy25) [inline EIAClientTest.java]: excited to work with junit

**Comment by galxy25** [comment]
other than my question about what types of exceptions we need to worry about catching, LGTM(Looks Good To Me)

**Comment by pault2k14** [comment]
No, there are others that we will need to catch depending on which API call we are making. Some of these API calls can throw several different exceptions. Our test cases will help us decide what to do with them, I'm currently working on the test cases that I took on Trello. If you go to http://commons.apache.org/proper/commons-net/apidocs/org/apache/commons/net/SocketClient.html#connect(java.net.InetAddress) and click on the name of the methods you can see the possible exceptions for each.

**Comment by galxy25** [comment]
@mkirby111 where do we stand with this branch?

**Comment by scottfabini** [comment]
Since this will be difficult to merge, I want to kill this branch, but not sure how.  All these changes have been incorporated into my latest "ScottTestCases" pull request, which I just submitted.

**Comment by galxy25** [comment]
@scottfabini You should just be able to hit the close and comment button at the bottom of this page. Then you can delete your branch.

**Comment by scottfabini** [comment]
Closing pull request

**Pull request closed without merging**